kernel/src/Plugins/FakePkg/FakePkgManager.hpp

```diff
@@ -26,6 +31,9 @@ namespace Mira
             Utils::Hook* m_SceSblKeymgrInvalidateKey;
             Utils::Hook* m_SceSblPfsSetKeysHook;
 
+            eventhandler_entry* m_processStartEvent;
+            eventhandler_entry* m_resumeEvent;
```

Comment:
m_processResumeEvent maybe a better name?

Reply:
It's not really the process ^^ it's more the PS4 itself